gci/models.py

```diff
@@ -1 +1,70 @@
-# Create your models here.
+from django.db import models
+
+
+class Students(models.Model):
+    student_id = models.IntegerField(default=0, primary_key=True)
+    student_name = models.CharField(max_length=100, default='')
+    program_year = models.ForeignKey('ProgramYear', null=True)
+
+    def __str__(self):
+        return self.student_name
+
+class Organizations(models.Model):
+    org_id = models.IntegerField(default=0, primary_key=True)
+    org_name = models.CharField(max_length=100, default='')
+    org_logo = models.ImageField(upload_to='', blank=True, default='')
+    org_url = models.URLField(default='', blank=True)
+    org_summary = models.TextField(max_length=300, default='')
+    students = models.ManyToManyField(Students)
+
+    def __str__(self):
+        return self.org_name
+
+class Tasks(models.Model):
+    name = models.CharField(max_length=200, default='')
+    description = models.TextField(max_length=500, default='')
+    status = models.IntegerField(default=1)
+    max_instances = models.IntegerField(default=1)
+    mentors = models.ManyToManyField('Mentors')
+    tags = models.ManyToManyField('Tags')
+    is_beginner = models.BooleanField(default=False)
+    categories = models.ManyToManyField('Categories')
+    time_to_complete_in_days = models.IntegerField(default=3)
+    external_url = models.URLField()
+    private_metadata = models.CharField(max_length=80, default='')
+    last_modified = models.DateField()
+    _orgs = models.ForeignKey(Organizations)
+
+    def __str__(self):
+        return self.name
+
+class TaskInstance(models.Model):
+    task_definition_name = models.ForeignKey(Tasks)
+    student_display_name = models.ForeignKey(Students)
+    organization_name = models.ForeignKey(Organizations)
```

Comment:
@jayvdb , I was having problem here, what about the fields like task_definition_id, student_display_id and organization_id? I don't know how to get ids from each model.

Reply:
You could rather rename these fields to `task`, `student` and `organization`. Because a `ForeignKey` attribute actually only holds the primary key of the related members. Check the docs [here](https://docs.djangoproject.com/en/1.11/ref/models/fields/#database-representation) for more on its internal representation.

Reply:
And please use `models.CASCADE` for the `on_delete` keyword argument for `ForeignKey`s.